enclave_manager/avalon_enclave_manager/avalon_enclave_info.py

```diff
@@ -334,6 +335,7 @@ def get_enclave_public_info(self):
         """
         Return information about the enclave
         """
+        signup_obj = enclave.SignupInfoSingeton()
```

Comment:
Better to name this variable consistently either as signup_obj or signup_cpp_obj.

Reply:
Ok, will rename.